tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/OVERVIEW.md

```diff
@@ -0,0 +1,57 @@
+# Holocene Hardfork - SystemConfig Upgrade
+Upgrades the `SystemConfig.sol` contract for Holocene.
+
+The batch will be executed on chain ID `11155111`, and contains `3` transactions.
```

Comment:
TODO: There are more transactions now since we made it a multi chain task.

Reply:
[0916f19](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/0916f198c3cfab43c3ba83d1f73b528fb7895e77)